packages/select/test/visual/lumo/select.test.js

```diff
@@ -46,6 +46,12 @@ describe('select', () => {
     await visualDiff(div, 'readonly');
   });
 
+  it('flex', async () => {
+    div.style.display = 'inline-flex';
+    div.style.height = '200px';
+    await visualDiff(div, 'flex');
+  });
```

Comment:
The `packages/select/test/visual/material/screenshots/select/baseline/flex.png` doesn't look right.. compared to all other screenshots. The field doesn't take the full width like e.g. the lumo variant.

Reply:
Good catch, thanks. I will check the Material version.